web-registry/src/components/organisms/CreditBatches/CreditBatches.tsx

```diff
@@ -58,71 +65,39 @@ const headCells: HeadCell[] = [
   { id: 'projectLocation', numeric: false, label: 'project location' },
 ];
 
-const useStyles = makeStyles((theme: Theme) => ({
-  section: {
-    [theme.breakpoints.up('md')]: {
-      paddingBottom: theme.spacing(22.25),
-    },
-    [theme.breakpoints.down('sm')]: {
-      paddingTop: theme.spacing(17.5),
-      paddingBottom: theme.spacing(17.5),
-    },
-  },
-  title: {
-    paddingBottom: theme.spacing(7.5),
-  },
-  tableBorder: {
-    border: `1px solid ${theme.palette.info.light}`,
-    borderRadius: 8,
-  },
-  wrap: {
-    whiteSpace: 'normal',
-    '& span': {
-      width: 138,
-    },
-  },
-  noWrap: {
-    whiteSpace: 'nowrap',
-  },
-  grey: {
-    color: theme.palette.info.main,
-  },
-  tableBody: {
-    backgroundColor: theme.palette.primary.main,
-  },
-}));
-
 const CreditBatches: React.FC<CreditBatchProps> = ({
   creditClassId,
-  projectPage = false,
+  filteredColumns,
+  withSection = false,
   creditBatches,
   titleAlign = 'center',
+  onTableChange,
```

Comment:
rename and add props to make the component more reusable and less linked to project page.